tools/occlum

```diff
@@ -248,6 +248,10 @@ cmd_start() {
     if [ "`get_enclave_debuggable_flag`" == "false" ]; then
         export OCCLUM_RELEASE_ENCLAVE=1
     fi
+
+    # /run/occlum is needed to save the occlum socket files for exec
+    mkdir -p /run/occlum
```

Comment:
You sure a normal, non-root user (without sudo) can create directories under `/run`?

Reply:
Seems that "/tmp" is still the best choice.